src/util/util.cpp

```diff
@@ -694,6 +694,65 @@ bool pointInsidePolygon(vector<vec2>& poly, vec2 p) {
 	return inside;
 }
 
+#define DATA_OFFSET_OFFSET 0x000A
+#define WIDTH_OFFSET 0x0012
+#define HEIGHT_OFFSET 0x0016
+#define BITS_PER_PIXEL_OFFSET 0x001C
+#define HEADER_SIZE 14
+#define INFO_HEADER_SIZE 40
+#define NO_COMPRESION 0
+#define MAX_NUMBER_OF_COLORS 0
+#define ALL_COLORS_REQUIRED 0
+
+void WriteImage(std::string fileName, BYTE* pixels, int width, int height, int bytesPerPixel)

```

Comment:
Can rename to the type of image? I'm guessing `WriteBMP`

Reply:
Fixed